example/index.html

```diff
@@ -1,17 +1,35 @@
 <!DOCTYPE html>
-<head>
-  <title>react-p5-wrapper</title>
+<head lang="en">
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>@P5-wrapper/react</title>
 </head>
 <body>
   <div class="container">
-    <h1>react-p5-wrapper</h1>
+    <h1>@P5-wrapper/react</h1>
     <h2>
-      <a href="https://github.com/P5-wrapper/react">View project on GitHub</a>
+      <a
+        href="https://github.com/P5-wrapper/react"
+        target="_blank"
```

Comment:
Update links to open in a new tab with security measures using `rel` added.

Reply:
👍